R/archive.R

```diff
@@ -275,19 +282,72 @@ new_epi_archive <- function(
     geo_type,
     time_type,
     other_keys,
-    compactify,
     clobberable_versions_start,
-    versions_end,
-    compactify_tol = .Machine$double.eps^0.5) {
+    versions_end) {
```

Comment:
question: I might've started in on this review too soon, but we should have the `compactify_abs_tol` and `compactify` args here, right?

Reply:
No, sorry, I just realized a couple of things after thinking it was ready + missed your comments (and one of your two comments didn't load when I just loaded the page?  Thought I freshly navigated to the URL...).

Yep, `new_epi_archive()` is supposed to be lower-level, basically as described [here](https://adv-r.hadley.nz/s3.html) or conventions similar to it, except their pass-subclass-to-lie-about-being-without-any-checks pattern doesn't really make sense to me and we really aren't set up for subclassing anyway.